### ISSUE TITLE
[Bug]: TypeError when converting axis to tuple. Not the correct way to initialise a tuple

### ISSUE THREAD

**Issue by iammarco11** [unifyai/ivy#14977]
### Bug Explanation

I encountered a TypeError while attempting to convert the `axis` variable to a tuple. The error occurred because `axis` was initially assigned an integer value, which is not iterable. [Link](https://github.com/unifyai/ivy/blob/master/ivy/functional/backends/jax/linear_algebra.py#L199) 

### Steps to Reproduce Bug

Call the `matrix_norm` function and pass `axis` as any integer.

### Environment

Windows 11, VSCode, conda python interpreter. 

### Ivy Version

Latest master

### Backend

- [ ] TensorFlow
- [ ] NumPy
- [ ] PyTorch
- [X] JAX
